onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -351,6 +353,45 @@ void BasicBackend::SetNumThreads(ov::AnyMap& device_config) {
     device_config.emplace(ov::inference_num_threads(session_context_.num_of_threads));
 }
 
+DynamicFlags BasicBackend::classify_shape_flags(const ov::CompiledModel& model) {
+  DynamicFlags flags;
+  for (const auto& input : model.inputs()) {
+    auto shape = input.get_partial_shape();
+    for (const auto& dim : shape) {
+      if (dim.is_dynamic()) {
+        flags.is_static = false;
+        std::cout << dim.get_min_length() << " , max = " << dim.get_max_length() << std::endl;
```

Comment:
This can be removed or can be accomodated with debug flag
```suggestion
        if (onnxruntime::openvino_ep::backend_utils::IsDebugEnabled()) {
            std::cout << "Min Dimension length:\t" << dim.get_min_length() << " , Max Dimension length:\t" << dim.get_max_length() << std::endl;
        }
```